### PR TITLE
Add Node.js code examples

### PR DESCRIPTION
**Pull request by matteofigus**
*Issue #5*

*Description of changes:*
Added Node.js code examples

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by juneb** [comment]
Wow! Thank you so much! I'm going to grab our Node developer do the required review. If everything looks good, we'll merge and publish.

**Comment by seebees** [comment]
Thanks for the contribution.  I doubt that I will have material feedback today, but I will get back to you.

**Comment by matteofigus** [comment]
Pleasure. Anything you want me to change, let me know 👍

**Comment by juneb** [comment]
@matteofigus I see some minor things. If you wouldn't mind updating, that would be great.
* To make it consistent with the other examples, please name the variable that contains the KMS client **kmsClient** (or whatever capitalization is common for this language). 

* It looks like we're missing an example for [encrypt](https://docs.aws.amazon.com/kms/latest/developerguide/programming-encryption.html#encryption) in Node.js. Can you please add it?

Again, I am truly grateful for the contribution. I'm sure we have community members using Node who will benefit from this code.

**Comment by juneb** [comment]
Can you please help me with indent spacing, too? What's typical in Node.js? I see variable declarations flush left and one space in. We should pick a style and be consistent.

File: doc_source/programming-client.md [hunk 1]
```diff
@@ -55,4 +55,9 @@ $KmsClient = new Aws\Kms\KmsClient([
 ]);
 ```
 
-------
\ No newline at end of file
+------
+#### [ Node.js ]
+
+```js
+const kms = new AWS.KMS({ apiVersion: '2014-11-01' });
```
Comment (seebees) [inline doc_source/programming-client.md]: The `apiVersion` is optional.  I would not use it here.

File: doc_source/programming-key-policies.md [hunk 2]
```diff
@@ -96,6 +96,24 @@ $result = $KmsClient->listKeyPolicies([
...
+
+For details, see the [ListKeyPolicies property](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/KMS.html#listKeyPolicies-property) in the *AWS SDK for Node.js*\.
+
+```js
+// List key policies
+//
+// Replace the following fictitious CMK ARN with a valid CMK ID or ARN
+
+const KeyId = 'arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab';
+const Limit = 10;
```
Comment (seebees) [inline doc_source/programming-key-policies.md]: @juneb PHP has a limit, but Ruby, Python et. al. don't.  I would default to having people put in a limit.  Does this mean we should update the other languages?

Comment (juneb) [reply doc_source/programming-key-policies.md]: I agree that Limit is a best practice, but given that this API returns only *default*, it's not needed here. Actually, I bet they added this example to show people that when they use this API and always get back *default* and nothing else, it's intentional. They haven't made a mistake.

Let me check with the writer for the PHP SDK. Unless it's critical to PHP for some reason, we should remove it and omit it from the Node.js.

Comment (juneb) [reply doc_source/programming-key-policies.md]: Good catch!

No, I don't think so. Limit is a best practice for List APIs, but this one always returns only *default*. I bet they added this example to reassure folks that when they always get *default*, they haven't made a mistake. 

I'll check with the PHP writer. If this isn't critical to the PHP code for some reason specific to PHP, I'll remove it from the PHP example. We shouldn't include it in the Node.js example.

File: doc_source/programming-client.md (hunk 1 above)
Comment (matteofigus) [reply doc_source/programming-client.md]: ok done

File: doc_source/programming-key-policies.md (hunk 2 above)
Comment (matteofigus) [reply doc_source/programming-key-policies.md]: I removed the limit from here but not PHP's. Going to do a separate PR for that so we don't get confused. Also, note that I removed the limit only on this specific call for node.js. Hopefully, that's ok, if you want me to remove it from other calls too let me know no problem 👍

**Comment by juneb** [comment]
@seebees Can you please review [PR 9](https://github.com/awsdocs/aws-kms-developer-guide/pull/9)? I'm swamped today, but I'll try to review tonight or tomorrow morning.

**Comment by juneb** [comment]
If there are no objections (@seebees ?), I'll merge this PR tonight.

@matteofigus Just FYI. To get this into the official online docs, I'll copy the changes into my XML source docs, have them reviewed again (yes, we do it again), publish online, and then refresh the markdown files in this repo. It takes a few days. If you have any questions, feel free to ask. Thanks again for everything!

**Comment by seebees** [comment]
@juneb Nope, this look good.  @matteofigus Thanks for taking the time it is really helpful to get documentation contributions!

**Comment by matteofigus** [comment]
Pleasure. I hope I'll be able to contribute more in the future.

**Pull request merged**